packages/docs/src/en/essentials/installation.md

```diff
@@ -30,7 +30,7 @@ This is by far the simplest way to get started with Alpine. Include the followin
 
 > Don't forget the "defer" attribute in the `<script>` tag.
 
-Notice the `@v2.x.x` in the provided CDN link. This will pull the latest version of Alpine version 2. For stability in production, it's recommended that you hardcode the latest version in the CDN link.
+Notice the `@3.x.x` in the provided CDN link. This will pull the latest version of Alpine version 2. For stability in production, it's recommended that you hardcode the latest version in the CDN link.
```

Comment:
```suggestion
Notice the `@3.x.x` in the provided CDN link. This will pull the latest version of Alpine version 3. For stability in production, it's recommended that you hardcode the latest version in the CDN link.
```

Reply:
Very detailed👍

Reply:
Sorry - I saw the "Commit suggestion" button too late, I'm not that firm in PRs and stuff... I made the change in my own PR and commited that.